docs/content/blog/a-year-of-releases-in-review.md

```diff
@@ -0,0 +1,100 @@
+---
+title: A year of releases in review
+description: >-
+  Over the past year, Keystone has received numerous updates, new features, and
+  essential bug fixes. Here’s a snapshot of the key improvements over the past
+  12 months.
+publishDate: 2024-08-07
+authorName: The Keystone Team
+authorHandle: https://twitter.com/keystonejs
+---
+Over the past year, Keystone has received numerous updates, new features, and essential bug fixes. Here’s a snapshot of the key improvements over the past 12 months.
+
+## Major Enhancements
+
+### Enhanced Database Integration
+
+One of the standout improvements this year is the update to Prisma `v5.13.0`. This update brings better performance and more robust features for database interactions. Alongside this, Keystone now supports extending the `PrismaClient`, providing more flexibility and control over database operations.
+
+### Interactive Transactions
+
+Keystone introduced `context.transaction`, allowing for interactive transactions within the `context`. This feature simplifies managing transactions, ensuring data integrity and consistency across operations.
+
+### Improved Migration Commands
+
+New migration commands like `keystone migrate create` and `keystone migrate apply` were added, making database migrations more straightforward and user-friendly.
+
+### Validation and Hooks
```

Comment:
This wasn't just [fields](https://github.com/keystonejs/keystone/pull/9057), but also [lists](https://github.com/keystonejs/keystone/pull/9056).

It may be worth expanding this, or wording more along the lines of "overhaul" to improve developer experience? Is it an addition if the functionality is the same?

I would also suggest a link to the Hooks guide, but that needs to be tidied up to match the new approach (something I was considering doing).